File: aulas/AppMinhaIdeiaDB/app/src/main/java/aluno/gabriel/appminhaideiadb/datamodel/ProdutoDataModel.java
package aluno.gabriel.appminhaideiadb.datamodel;

public class ProdutoDataModel {
    public static final String DB_TABLE = "PRODUTO";
    public static final String ID = "id";
    public static final String NAME = "nome";
    public static final String FABRICANTE = "fabricante";
    public static String query = "";

    public static String queryCreateTable(){
        query += "CREATE TABLE IF NOT EXISTS "+DB_TABLE+" ( ";
        query += ID+" INTEGER PRIMARY KEY AUTOINCREMENT, ";
        query += NAME+" TEXT NOT NULL, ";
        query += FABRICANTE+" TEXT NOT NULL ";
        query += " ) ";
        return query;
    }
}

File: treinos/treino009/app/src/main/java/aluno/gabriel/treino009/datamodel/ProdutoDataModel.kt
package aluno.gabriel.treino009.datamodel

import aluno.gabriel.treino009.core.ApplicationUtil

class ProdutoDataModel {
    companion object{ @JvmStatic
        fun queryCreateTable(): String {
        ApplicationUtil.query += "CREATE TABLE IF NOT EXISTS "+ApplicationUtil.TABLE_NAME+" ( "
        ApplicationUtil.query += ApplicationUtil.ID+" INTEGER PRIMARY KEY AUTOINCREMENT, "
        ApplicationUtil.query += ApplicationUtil.NOME+" TEXT NOT NULL, "
        ApplicationUtil.query += ApplicationUtil.FABRICANTE+" TEXT NOT NULL, "
        ApplicationUtil.query += ApplicationUtil.QUANTIDADE+" INTEGER DEFAULT 0, "
        ApplicationUtil.query += ApplicationUtil.PRECO+" REAL DEFAULT 0.00 "
        ApplicationUtil.query += " ) "
        return ApplicationUtil.query
        }


    }
}